packages/web-components/docs/markdown-contents.md

```diff
@@ -1,9 +1,87 @@
-# Markdown contents
+## Markdown contents
 
-`<dds-content-*-copy>` components, used as the copy text of our content pattern, takes markup as its content. It takes either `content` property (_not_ attribute) or child text node as the markdown content. The markdown content will be converted to HTML with sanitization.
+In most cases, `<dds-content-*>` is used with its "copy" content. There are several ways to specify the copy content.
+
+### Using `<dds-content-*-copy>`
+
+`<dds-content-*-copy>` are Web Components that automatically renders the given markdown content to HTML. It takes either `content` property (_not_ attribute) or child text node as the markdown content. The markdown content will be converted to HTML with sanitization.
 
 Given the nature of child text node, some extra caution is required to use child text node as the markdown content:
 
 * After-initialization change of markdown content via child text node is not supported.
 * Don't put any extra whitespace, e.g. line feeds, between the start/end tag and the markdown content.
 * HTML-escape the content as needed.
+
+### Rendering markdown on server
+
+While `<dds-content-*-copy>` provides an easy way to use markdown for `<dds-content-*>`, it requires markdown parser and HTML sanitizer has to be downloaded and run in browser. To reduce the cost of downloading and running, especially if the target network and device environment is limiting, rendering markdown on server is often helpful.
+
+For example, a Handlebars helper that works with our [`markdownToHtml` utility](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js) can be defined to convert generate HTML from markdown content:
```

Comment:
```suggestion
For example, a Handlebars helper that works with the Carbon for IBM.com [`markdownToHtml` utility](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js) can be defined to convert generated HTML from markdown content:
```

Reply:
Remember, no "you" "yours" "our", etc :)

Reply:
Thanks @jeffchew for catching this!